Rakefile

```diff
@@ -84,6 +84,32 @@ task :console do
   IRB.start
 end
 
+desc "Run acceptance tests"
+task :acceptance do |version, pe_version|
+  version ||= '1.9.2.0'
+  pe_version ||= '2018.1'
+  cmd = <<HEREDOC
+export BUNDLE_PATH=.bundle/gems
+export BUNDLE_BIN=.bundle/bin
+# We should change this to SHA and SUITE_VERSION in the acceptance tests
+export PE_RAZOR_SERVER_PACKAGE_BUILD_VERSION=#{version}
+
+bundle install
+
+beaker-hostgenerator --pe_dir=http://enterprise.delivery.puppetlabs.net/#{pe_version}/ci-ready \
+--disable-default-role --hypervisor vmpooler centos7-64mdc-64a > hosts.cfg
+
+bundle exec beaker --xml --debug --root-keys --repo-proxy --hosts hosts.cfg \
+--type pe --keyfile ~/.ssh/id_rsa-acceptance --preserve-hosts onfail \
+--helper lib/helper.rb --pre-suite suites/pre_suite/install-server-from-module \
+--tests suites/tests/smoke,suites/tests/full --load-path lib
+HEREDOC
```

Comment:
This is an interesting way to do this. I guess it makes sense (trying to manage the environment for each command etc. isn't gonna be fun in Ruby). Took me a moment to /convince/ myself it's the best option, but I don't have any better ideas